src/pattern-library/components/LibraryHome.js

```diff
@@ -0,0 +1,37 @@
+import Library from './Library';
+
+export default function LibraryHome() {
+  return (
+    <Library.Page title="Pattern Library">
+      <p>
+        This pattern library demonstrates foundations, patterns and components
+        that can be used in Hypothesis front-end applications.
+      </p>
+
+      <h2 className="LibraryPage__subheading">Foundations</h2>
+
+      <p>
+        <strong>Foundations</strong> are the core design system elements—colors,
+        spacing systems, typography—that define the underlying design
+        fundamentals in Hypothesis UI.
+      </p>
+
+      <p>They exist independently of implementation.</p>
+
+      <h2 className="LibraryPage__subheading">Patterns</h2>
+
+      <p>
+        <strong>Patterns</strong> are modular implementations of the design
+        system <strong>foundations</strong>—from the atomic to the complex.
+        These implementations are in HTML and CSS.
+      </p>
+
+      <h2 className="LibraryPage__subheading">Components</h2>
+
+      <p>
+        <strong>Components</strong> are preact components that are built using
```

Comment:
```suggestion
        <strong>Components</strong> are Preact components that are built using
```

Reply:
Interesting, thank you! I had been under the mistaken impression all along that _Preact_ was not capitalized.